### PR TITLE
fix: Aggregating indices now allow multiple field names in the key column

### PR DESCRIPTION
**Pull request by ericf-firebolt**
Resolves https://packboard.atlassian.net/browse/FIR-8715

### Checklist

- [x] I have run this code in development and it appears to resolve the stated issue.
- [x] This PR includes tests, or tests are not required/relevant for this PR.
- [x] I have updated `CHANGELOG.md` and added information about my change.
- [x] If this PR requires a new PyPI release I have bumped the version number.
- [x] I have pulled/merged from the main branch if there are merge conflicts.
- [x] I have verified that this PR contains only code changes relevant to this PR.

### CODE REVIEW

**Comment by octavianzarzu30** [comment]
@stepansergeevitch can you please review?

**Review by octavianzarzu30 — APPROVED** [review]
lgtm

File: dbt/adapters/firebolt/impl.py [hunk 1]
```diff
@@ -19,22 +19,28 @@
...
-        example index name: my_model_customer_id_join_1633504263.
+        Name an index according to the following format, joined by `_`:
+        index type, relation name, key/join column, timestamp (unix & UTC)
+        example index name: join_my_model_customer_id_1633504263.
         """
         now_unix = time.mktime(datetime.utcnow().timetuple())
-        spine_col = self.key_column if self.key_column else self.join_column
-        inputs = [relation.identifier, spine_col, str(self.type), str(int(now_unix))]
-        string = '__'.join(inputs)[0:254]
+        spine_col = '_'.join(self.key_column if self.key_column else self.join_column)
```
Comment (stepansergeevitch) [inline dbt/adapters/firebolt/impl.py:35]: nit:
```
spine_col = '_'.join(self.key_column or self.join_column)
```

**Review by stepansergeevitch — APPROVED** [review]
lgtm

**Pull request merged**